docs/xml/elements/street_segment.rst

```diff
@@ -6,106 +6,107 @@ geography (i.e., segment) is contained within. The start address house number mu
+|                      |                                         |              |            |                        |                        |
+|                      |                                         |              |            |                        |                        |
+|                      |                                         |              |            |                        |                        |
+|                      |                                         |              |            |                        |                        |
+|                      |                                         |              |            |                        |                        |
+|                      |                                         |              |            |                        |                        |
+|                      |                                         |              |            |                        |                        |
++----------------------+-----------------------------------------+--------------+------------+------------------------+------------------------+
+| IncludesAllAddresses | xs:boolean                              | Optional     | Single     |Specifies if the segment|If the field is not     |
+|                      |                                         |              |            |covers every address on |present or invalid, the |
+|                      |                                         |              |            |this street. If this is |implementation is       |
+|                      |                                         |              |            |*true*, then the values |required to ignore it.  |
+|                      |                                         |              |            |of **StartHouseNumber** |                        |
+|                      |                                         |              |            |and **EndHouseNumber**  |                        |
+|                      |                                         |              |            |should be ignored. The  |                        |
+|                      |                                         |              |            |value of **OddEvenBoth**|                        |
+|                      |                                         |              |            |must be *both*.         |                        |
+|                      |                                         |              |            |                        |                        |
++----------------------+-----------------------------------------+--------------+------------+------------------------+------------------------+
+| OddEvenBoth          |:doc:`OebEnum <../enumerations/oeb_enum>`| Optional     | Single     |Specifies whether the   |If the field is not     |
```

Comment:
Is `OddEvenBoth` now optional with the new flag? I would suggest that it still be **Required** as I could imagine a circumstance where you could specify the entire odd or even side of the street with the new flag. If that's too error prone, the Error Handling column needs to be updated.